### ISSUE TITLE
URI malformed

### ISSUE THREAD

**Issue by rkhomi** [cypress-io/cypress#22529]
### Current behavior

Getting randomly this error in CI env

URIError: The following error originated from your test code, not from Cypress.

  > URI malformed

When Cypress detects uncaught errors originating from your test code it will automatically fail the current test.

Cypress could not associate this error to any specific test.

We dynamically generated a new test to display this failure.
    at decodeURIComponent (<anonymous>)
    at Object.decodeBase64Unicode (https://my_app/__cypress/runner/cypress_runner.js:187982:12)
    at base64toJs (https://my_app/__cypress/runner/cypress_runner.js:187199:76)
    at Object.extractSourceMap (https://my_app/__cypress/runner/cypress_runner.js:187157:21)
    at extractSourceMap (https://my_app/__cypress/runner/cypress_runner.js:186237:87)
    at tryCatcher (https://my_app/__cypress/runner/cypress_runner.js:13022:23)
    at MappingPromiseArray._promiseFulfilled (https://my_app__cypress/runner/cypress_runner.js:10142:38)
    at MappingPromiseArray.PromiseArray._iterate (https://my_app/__cypress/runner/cypress_runner.js:11344:31)
    at MappingPromiseArray.init (https://my_app/__cypress/runner/cypress_runner.js:11308:10)
    at Promise._settlePromise (https://my_app/__cypress/runner/cypress_runner.js:11011:21)
    at Promise._settlePromise0 (https:/my_app/__cypress/runner/cypress_runner.js:11059:10)
    at Promise._settlePromises (https://my_app/__cypress/runner/cypress_runner.js:11139:18)
    at Promise._fulfill (https://my_app/__cypress/runner/cypress_runner.js:11083:18)
    at MappingPromiseArray.PromiseArray._resolve (https://my_app/__cypress/runner/cypress_runner.js:11356:19)
    at MappingPromiseArray._promiseFulfilled (https://my_app__cypress/runner/cypress_runner.js:10182:18)
    at Promise._settlePromise (https://my_app/__cypress/runner/cypress_runner.js:11019:26)

### Desired behavior

This issue should not occur

### Test code to reproduce

No code to reproduce since it happens randomly

### Cypress Version

9.6.1

**Comment by rkhomi** [comment]
> What is the name of your spec file and what OS are you running on?

That spec has correct name, just a regular patter my-featture.spec.ts (it's not exact name just example)
Also that spec passed on retry and failed only once for a long time

I run my testss insidee cypress/browsers:node14.17.6-chrome100-ff98

**Comment by rkhomi** [comment]
```
const id = uuid();
	on('file:preprocessor', file => {
		file.outputPath = file.outputPath.replace(/^(.*\/)(.*?)(\..*)$/, `$1$2.${id}$3`);
		return filePreprocessor(file);
	});
```

However I am using this in plugins.ts due to this issue https://davidwcai.medium.com/fix-cypress-invalid-or-unexpected-token-errors-in-parallel-runs-e887d7efa945

I guess that has smth to do with this error

**Comment by tbiethman** [comment]
@rkhomi I know you reported this issue as being intermittent, but can you try putting together a minimal reproduction? It'd be helpful to see the cypress [debug logs](https://docs.cypress.io/guides/references/troubleshooting#Print-DEBUG-logs) around this error.

**Comment by tbiethman** [comment]
Unfortunately we have to close this issue due to inactivity. Right now there doesn't seem to be enough information to reproduce the problem on our end. We'll have to close this issue until we can reproduce it. This does not mean that your issue is not happening - it just means that we do not have a path to move forward.

Please open a new issue with a reproducible example and link to this issue. Here are some tips for providing a [Short, Self Contained, Correct, Example](http://sscce.org/) and our own [Troubleshooting Cypress](https://on.cypress.io/troubleshooting) guide.
